qa/rpc-tests/txoutsbyaddress.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+
+class TxOutsByAddressTest(BitcoinTestFramework):
+    """Tests -txoutsbyaddressindex"""
+
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 3
+
+    def setup_network(self, split=False):
+        print("Setup network...")
+        self.nodes = []
```

Comment:
```py
self.nodes = start_nodes(3, self.options.tmpdir, self.extra_args)
```
should work as well. (You can put `self.extra_args = [['-txoutindex'], []. []]` in `__init__()`)